tests/node/rpc/structs/ImageTests.js

```diff
@@ -0,0 +1,45 @@
+const SDL = require('./../../../../lib/js/dist/SDL');
+const Image = SDL.rpc.structs.Image;
+
+const Test = require('./../../../Test.js');
+const Validator = require('./../../../Validator.js');
+const assertTrue = Validator.assertTrue.bind(Validator);
+const assertEquals = Validator.assertEquals.bind(Validator);
+const assertNull = Validator.assertNull.bind(Validator);
+const assertNullOrUndefined = Validator.assertNullOrUndefined.bind(Validator);
+const assertNotNull = Validator.assertNotNull.bind(Validator);
+
+let msg;
```

Comment:
This looks inconsistent with the other files with how they declare `msg` from within the tests

Reply:
Added BaseStructTests which acts similarly to BaseRpcTests.